packages/flutter/test/widgets/animated_grid_test.dart

```diff
@@ -200,12 +200,12 @@ void main() {
         ),
       );
 
-      double itemScale(int index) =>
-          tester.widget<ScaleTransition>(find.byKey(ValueKey<int>(index), skipOffstage: false)).scale.value;
-      double itemLeft(int index) => tester.getTopLeft(find.byKey(ValueKey<int>(index), skipOffstage: false)).dx;
-      double itemRight(int index) => tester.getTopRight(find.byKey(ValueKey<int>(index), skipOffstage: false)).dx;
+      Finder finderOfIndex(int index) => find.byKey(ValueKey<int>(index), skipOffstage: false);
```

Comment:
The PR description mentioned this is not a breaking change, would you mind reverting the changes to existing tests so we know we have not introduced a regression?

Reply:
Sure! Don't I need to add tests if there is no breaking change?